src/cplscheme/tests/AbsoluteOrRelativeConvergenceMeasureTest.cpp

```diff
@@ -0,0 +1,41 @@
+#include <Eigen/Core>
+#include "../impl/AbsoluteOrRelativeConvergenceMeasure.hpp"
+#include "cplscheme/impl/ConvergenceMeasure.hpp"
+#include "testing/TestContext.hpp"
+#include "testing/Testing.hpp"
+
+using namespace precice;
+using namespace cplscheme;
+
+BOOST_AUTO_TEST_SUITE(CplSchemeTests)
+
+BOOST_AUTO_TEST_CASE(AbsoluteOrRelativeConvergenceMeasureTest)
+{
+  PRECICE_TEST(1_rank);
+  using Eigen::Vector3d;
+  // Create convergence measure for Vector data
+  double                                                convergenceLimit        = 1.0;
+  double                                                convergenceLimitPercent = 0.1;
```

Comment:
```suggestion
  double                                                absLimit = 1.0;
  double                                                relLimit = 0.1;
```
To use the same names as in the configuration.